### ISSUE TITLE
How to avoid reading dot files?

### ISSUE THREAD

**Issue by appernetic** [mihneadb/node-directory-tree#7]
Which would be the best approach to avoid reading dot files? 

Can I use the filter for that and if, how do I use it for that?

First I thought it was because I used the dotenv module that it read the dot files, but now when I have changed how the express app is reading its initial configuration and removed dotenv node.directory-tree is still reading dot files. 

I am developing on a Mac so this could be OSX related when I get a .DS_Store file in the tree. I have left to test it in my Linux dev server to see if I get rid of this problem.

**Comment by mihneadb** [comment]
You can't perform such a filter with the current implementation. You can tell it to find files that contain a given substring, but you can't tell it to exclude files matching a pattern. PR welcome if you figure out a nice way of implementing this :). I guess the best way to go would be some sort of include and exclude option that allow specifying regexes.

**Comment by appernetic** [comment]
I am back to this little problem again. I did a quick solution:

```javascript
}).filter(function (e) {
                return e != null;
            }).filter(function (e) {
                return e.path !== '.placeholder';
            });
```
I did not try to find dot files. In Ubuntu I do not have .DS_Store files but I need to have placeholder files so that git doesn't remove folders if I delete all files. So instead I leave a .placeholder file. Then I only have to find that file and filter. Easy peasy! 
This solution is not worthy a PR I think..
